Splash/Splash/ShowersViewController.m

```diff
@@ -44,14 +57,23 @@ -(void) getData {
           for (int i = objects.count - 1; i >= 0; i = i - 1) {
               [self.showersArray addObject:objects[i]];
           }
-          //self.showersArray = objects;
+          self.filteredData = self.showersArray;
           [self.showersTableView reloadData];
+          //self.showersArray = objects;
       } else {
         // Log details of the failure
         NSLog(@"Error: %@ %@", error, [error userInfo]);
```

Comment:
We talked about using advanced iOS techniques. Let's improve this further, you can do this as a separate pull request. 

NSLog will continue to print in consoles even after the app is shipped to the app store. For this not to happen, we can use a DEBUG flag and use that to print logs only when testing and not otherwise. 

Feel free to do some digging, if you face any issues please reachout.